lib/helpers/answerHelper.js

```diff
@@ -92,7 +101,13 @@ function summaryFormatter(answerObject) {
                         Object.values(answerObject[question])[0]
                     );
                 } else {
-                    answerIndex[question].value = multipleAnswersFormat(answerObject[question]); // A question with more than one answer appears in a multi-line format
+                    let answers = answerObject[question];
```

Comment:
If the simplyifyUiSchema goes this can become something like:

```
const uiSchema = fullUiSchema[question];

            if (uiSchema && uiSchema.options && uiSchema.options.outputOrder) {
                answers = JSON.parse(
                    JSON.stringify(answers, uiSchema.options.outputOrder)
                );
            }
```